_includes/cwl/metadata_example2.cwl

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+id: Example tool
+label: Example tool
+cwlVersion: v1.0
+doc: |
+    An example tool demonstrating metadata. Note that this is an example and the metadata is not necessarily consistent.
+
+requirements:
+  - class: ShellCommandRequirement
+
+inputs:
+  bam_input:
+    type: File
+    doc: The BAM file used as input
+    format: http://edamontology.org/format_2572
+    inputBinding:
+      position: 1
+
+stdout: output.txt
+
+outputs:
+  report:
+    type: File
+    format: http://edamontology.org/format_1964
+    outputBinding:
+      glob: "*.txt"
+    doc: A text file that contains a line count
+
+baseCommand: ["wc", "-l"]
+
+$namespaces:
+  s: https://schema.org/
+
+$schemas:
+- https://schema.org/docs/schema_org_rdfa.html
+
+s:author:
```

Comment:
is the assumption that the author is always a contributor, and therefore shouldn't be included also in the list of contributors?